cmd/list/account/cmd.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package account
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/output"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "account-roles",
+	Aliases: []string{"accountrole", "account-role", "accountroles"},
+	Short:   "List account roles and policies",
+	Long:    "List account roles and policies for the current AWS account.",
+	Example: `  # List all account roles
+  rosa list account-roles`,
+	Args: cobra.NoArgs,
```

Comment:
Do we want to support `--version` (or even `--prefix`) to filter the results? Is it even necessary?
/cc @arendej

Reply:
As a CLI tool, for now I think the list account-roles makes sense to rely on GNU tools for filtering.
- `--prefix` doesn't make a ton of sense yet because there's a lot of possibilities there.
- if we can be sure about the version for the roles, then `--version` makes sense to have.